dxr/plugins/rust/tests/__init__.py

```diff
@@ -8,8 +15,10 @@ class RustSingleFileTestCase(SingleFileTestCase):
     @classmethod
     def config_input(cls, config_dir_path):
         config = super(RustSingleFileTestCase, cls).config_input(config_dir_path)
-
+        
```

Comment:
Extra whitespace. (I'm going to clean up the whole codebase in the near future and put a little tooling in place to keep it that way—though not enough to be annoying.)